test/types.go

```diff
@@ -118,6 +118,7 @@ type VeleroConfig struct {
 	ServiceAccountNameToInstall       string
 	EKSPolicyARN                      string
 	FailFast                          bool
+	DisableVspherePlugin              bool
```

Comment:
Change it to "HasVSpherePlugin"?  And following "if" will only need to check this single flag.

Reply:
I agree that using the positive wording is more intuitive, but we cannot replace all the provider checking with this flag, for example:
``` go
						// If the CloudProvider is vSphere and don't need vSphere plugin,
						// Only install the AWS plugin.
						// If do nothing here, the default behavior is
						// installing both AWS and vSphere plugins.
						if OriginVeleroCfg.CloudProvider == Vsphere &&
							OriginVeleroCfg.DisableVspherePlugin {
							OriginVeleroCfg.Plugins = migrationNeedPlugins[AWS][0]
						}
```

It should be changed to something like the following.
The HasVspherePlugin flag is defaulted to false, so we need to check the provider to make sure it's for vSphere environment.

``` go
						// If HasVspherePlugin is false, only install the AWS plugin.
						// If do nothing here, the default behavior is
						// installing both AWS and vSphere plugins.
						if OriginVeleroCfg.CloudProvider == Vsphere &&
							!OriginVeleroCfg.HasVspherePlugin {
							OriginVeleroCfg.Plugins = migrationNeedPlugins[AWS][0]
						}
```